### DISCUSSION TITLE
Hide Info in Admin Panel

### DISCUSSION THREAD

**Discussion by WatermelonModders** [mastodon/mastodon#27628] [Q&A]
Hello, I recently started a Mastodon instance and I have a problem.

Mastodon lets me see basically a lot of info about the users that are signed up (IPs, Emails, etc.), and I don't want to be able to see this info. I believe it's an invasion of privacy, and I don't want me or my mods to be able to see that info. Is there any way to remove the ability to see that info?

**Comment by mjankowski** [comment]
An issue here is that in order to be able to do (some types of) moderation correctly, this information is sometimes needed.
